app/serializers/e_invoices/payments/common.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+module EInvoices
+  module Payments
+    module Common
+      def credits_and_payments(&block)
```

Comment:
Does this really need to be in the module ? It seems to me this is only used in one class ?